django/db/migrations/loader.py

```diff
@@ -64,6 +64,10 @@ def load_disk(self):
         self.unmigrated_apps = set()
         self.migrated_apps = set()
         for app_config in apps.get_app_configs():
+            if (self.connection is not None and
```

Comment:
As far as I can see you can instantiate the `MigrationLoader` in the questioner and writer with `connection=None` and `load=False` to achieve the same behavior there and drop the `@classmethod` from `migrations_module()`